opsml/projects/_run_manager.py

```diff
@@ -133,8 +171,11 @@ def _log_hardware_metrics(self, interval: int) -> None:
         assert self.active_run is not None, "active_run should not be None"
 
         # run hardware logger in background thread
-        executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
-        executor.submit(run_hardware_logger, interval, self.active_run)
+        queue: Queue[HardwareMetrics] = Queue()
```

Comment:
Realized we can assign queue without adding it to the `active run` or `run_manager`